.env.example

```diff
@@ -32,3 +32,5 @@ JS_PATH=js/min
 JS_IS_MIN=true
 
 LINK_PREVIEW_DEFAULT=http://dummyimage.com/200x200/ffa800&text=No+Preview
```

Comment:
Why didn't you just move it to a config file? It's not a variable that depends from environment.